fire/api/model/sagstyper.py

```diff
@@ -25,6 +25,10 @@ class EventType(enum.Enum):
     KOMMENTAR = 9
     GRAFIK_INDSAT = 10
     GRAFIK_NEDLAGT = 11
+    PUNKTGRUPPE_MODIFICERET = 12
+    PUNKTGRUPPE_NEDLAGT = 13
+    TIDSSERIE_MODIFICERET = 14
+    TIDSSERIE_NEDLAGT = 15
```

Comment:
Hvad dækker `MODIFICERET` over? Kan ikke huske, hvad det er, hvis vi har talt om det.

Reply:
Oprettet eller ændret. Sagsevents er her jo primært til ære for udstillingsmodellen og her skælnes ikke mellem om noget nyt er kommet til eller om noget eksisterende er ændret. Så i stedet for at slæbe rundt på tre typer: opret, ændr og nedlæg, så nøjes vi bare med modificeret og nedlagt.